recipe/cross-gfortran.linux-aarch64.sh

```diff
@@ -0,0 +1,108 @@
+export ompi_cv_fortran_have_INTEGERp8=yes
+export ompi_cv_fortran_have_LOGICAL=yes
+export ompi_cv_fortran_have_LOGICALp1=yes
+export ompi_cv_fortran_have_LOGICALp2=yes
+export ompi_cv_fortran_have_LOGICALp4=yes
+export ompi_cv_fortran_have_LOGICALp8=yes
+export ompi_cv_fortran_have_REAL=yes
+export ompi_cv_fortran_have_REALp16=yes
+export ompi_cv_fortran_have_REALp2=no
+export ompi_cv_fortran_have_REALp4=yes
+export ompi_cv_fortran_have_REALp8=yes
+export ompi_cv_fortran_have_bind_c_sub=yes
+export ompi_cv_fortran_have_bind_c_type=yes
+export ompi_cv_fortran_have_bind_c_type_name=yes
+export ompi_cv_fortran_have_iso_c_binding=yes
+export ompi_cv_fortran_have_iso_fortran_env=yes
+export ompi_cv_fortran_have_iso_fortran_env_real16=no
+export ompi_cv_fortran_have_storage_size=yes
+export ompi_cv_fortran_ignore_tkr_data='1:type(*), dimension(*):!GCC$ ATTRIBUTES NO_ARG_CHECK ::'
+export ompi_cv_fortran_interface=yes
+export ompi_cv_fortran_kind_value_0=0
+export ompi_cv_fortran_kind_value_C_DOUBLE=8
+export ompi_cv_fortran_kind_value_C_DOUBLE_COMPLEX=8
+export ompi_cv_fortran_kind_value_C_FLOAT=4
+export ompi_cv_fortran_kind_value_C_FLOAT_COMPLEX=4
+export ompi_cv_fortran_kind_value_C_INT16_T=2
+export ompi_cv_fortran_kind_value_C_INT32_T=4
+export ompi_cv_fortran_kind_value_C_INT64_T=8
+export ompi_cv_fortran_kind_value_C_INT=4
+export ompi_cv_fortran_kind_value_C_LONG_DOUBLE=16
+export ompi_cv_fortran_kind_value_C_LONG_DOUBLE_COMPLEX=16
+export ompi_cv_fortran_kind_value_C_LONG_LONG=8
+export ompi_cv_fortran_kind_value_C_SHORT=2
+export ompi_cv_fortran_kind_value_C_SIGNED_CHAR=1
+export ompi_cv_fortran_logical_array_correct=yes
+export ompi_cv_fortran_max_array_rank=15
+export ompi_cv_fortran_module_include_flag=-I
+export ompi_cv_fortran_optional=yes
+export ompi_cv_fortran_private=yes
+export ompi_cv_fortran_procedure=yes
+export ompi_cv_fortran_sizeof_CHARACTER=1
+export ompi_cv_fortran_sizeof_COMPLEX=8
+export ompi_cv_fortran_sizeof_COMPLEXp16=16
+export ompi_cv_fortran_sizeof_COMPLEXp32=32
+export ompi_cv_fortran_sizeof_COMPLEXp8=8
+export ompi_cv_fortran_sizeof_DOUBLE_COMPLEX=16
+export ompi_cv_fortran_sizeof_DOUBLE_PRECISION=8
+export ompi_cv_fortran_sizeof_INTEGER=4
+export ompi_cv_fortran_sizeof_INTEGERp16=16
+export ompi_cv_fortran_sizeof_INTEGERp1=1
+export ompi_cv_fortran_sizeof_INTEGERp2=2
+export ompi_cv_fortran_sizeof_INTEGERp4=4
+export ompi_cv_fortran_sizeof_INTEGERp8=8
+export ompi_cv_fortran_sizeof_LOGICAL=4
+export ompi_cv_fortran_sizeof_LOGICALp1=1
+export ompi_cv_fortran_sizeof_LOGICALp2=2
+export ompi_cv_fortran_sizeof_LOGICALp4=4
+export ompi_cv_fortran_sizeof_LOGICALp8=8
+export ompi_cv_fortran_sizeof_REAL=4
+export ompi_cv_fortran_sizeof_REALp16=16
+export ompi_cv_fortran_sizeof_REALp4=4
+export ompi_cv_fortran_sizeof_REALp8=8
+export ompi_cv_fortran_sizeof_type_test_mpi_handle_=4
+export ompi_cv_fortran_true_value=1
+export ompi_cv_fortran_use_only=yes
+export ompi_cv_real16_c_equiv=yes
```

Comment:
@jsquyres This is a very minor nit.. but `ompi_cv_real16_c_equiv` is the only Fortran-related variable required for cross compiling that does not use a `mpi_cv_fortran_` prefix (note the missing `fortran_` bit). Would it make sense to rename it to follow the pattern of all the others variables?